iml-wire-types/wire-types.rs

```diff
@@ -1212,6 +1212,84 @@ pub mod db {
         }
     }
 
+    /// Record from the `chroma_core_ostpool` table
+    #[derive(serde::Deserialize, serde::Serialize, Debug, Clone)]
+    pub struct OstPoolRecord {
+        pub id: u32,
+        pub name: String,
+        pub filesystem_id: u32,
+        pub not_deleted: Option<bool>,
```

Comment:
Probably `is_deleted` is preferable, since it has nice default (`false`, i.e. present) and doesn't require to perform double negation in `false` case.

However, if the field name clones the name from the other part of the system, then it is better to leave it as is.

Reply:
I agree 100%, but this naming convention predates me and is persistent throughout the API / db layer (which this type is simply representing.